sbncode/SinglePhotonAnalysis/SinglePhoton_module.cc

```diff
@@ -0,0 +1,1520 @@
+#include "SinglePhoton_module.h"
+#include "HelperFunctions/helper_processors.h" //now it can use variables defined in singlephoton.h
+#include "HelperFunctions/helper_PandoraPFParticles.h" //now it can use variables defined in singlephoton.h
+#include "Libraries/init_branches.h"
+#include "Libraries/reco_truth_matching.h"
+//#include "analyze_Template.h"
+#include "Libraries/analyze_OpFlashes.h"
+#include "Libraries/analyze_Tracks.h"
+#include "Libraries/analyze_Showers.h"
+#include "Libraries/analyze_MCTruth.h"
+#include "Libraries/analyze_EventWeight.h"
+#include "Libraries/analyze_Slice.h"
+#include "Libraries/analyze_Geant4.h"
+#include "Libraries/fiducial_volume.h"
+#include "Libraries/second_shower_search.h"
+#include "Libraries/isolation.h"
+#include "Libraries/output_ntuple.h"
```

Comment:
Please explicitly include `sbncode/SinglePhotonAnalysis/` on the front of all your includes.